bindings/rust/evmc-vm/src/lib.rs

```diff
@@ -419,6 +519,43 @@ impl<'a> ExecutionContext<'a> {
             )
         }
     }
+
+    /// Read from a transient storage key.
+    pub fn get_transient_storage(&self, address: &Address, key: &Bytes32) -> Bytes32 {
+        unsafe {
+            assert!(self.host.get_transient_storage.is_some());
+            self.host.get_transient_storage.unwrap()(
```

Comment:
I see a mixture of asserts and expects. I would stick to expect, but since other functions already use assert, we can keep it this way.

Reply:
Agreed. But as you said it is probably better to keep it consistent.